sphinx_gallery/gen_rst.py

```diff
@@ -481,6 +477,9 @@ def execute_script(code_block, example_globals, image_path, fig_count,
     time_elapsed = 0
     stdout = ''
 
+    if src_file in gallery_conf['failing_examples']:
```

Comment:
You should probably put a comment above this line, saying that if an error has already happened we don't want to keep executing block since only the first error matters.